pkg/sensors/tracing/kprobe_test.go

```diff
@@ -2300,3 +2305,90 @@ func TestKprobeMatchArgsFdPrefix(t *testing.T) {
 	err = jsonchecker.JsonTestCheck(t, checker)
 	assert.NoError(t, err)
 }
+
+func loadTestCrd(t *testing.T) error {
+	testHook := `apiVersion: cilium.io/v1alpha1
+kind: TracingPolicy
+metadata:
+ name: "sys-write"
+spec:
+ kprobes:
+ - call: "__x64_sys_write"
+   syscall: true
+`
+	var sens []*sensors.Sensor
+
+	cnf, _ := yaml.ReadConfigYaml(testHook)
+	if cnf != nil {
+		var err error
+		sens, err = sensors.GetSensorsFromParserPolicy(&cnf.Spec)
+		if err != nil {
+			return err
+		}
+	}
+
+	b := base.GetInitialSensor()
+	if err := b.Load(context.TODO(), option.Config.BpfDir, option.Config.MapDir, option.Config.CiliumDir); err != nil {
```

Comment:
Looks good to me though should we also unload the program here in the flow?

Reply:
Would also be good to replace context.TODO()s here with a single context.Background() like we recently did for other tests.

Reply:
the test logs show that the other program (sys_write kprobe) is already being unloaded at the end, so I think that part is fine..